neuromation/cli/job.py

```diff
@@ -599,6 +600,8 @@ def job() -> None:
             click.echo(job_resolved)
         except ValueError as e:
             errors.append((job, e))
+        except AuthorizationError as e:
+            errors.append((job, ValueError(f"Not enough permissions")))
```

Comment:
How to figure out what job has no permissions to be killed?

Reply:
It's resolved few lines after:
```python
    def format_fail(job: str, reason: Exception) -> str:
        return click.style(f"Cannot kill job {job}: {reason}", fg="red")
```

Reply:
```
$ neuro job kill job-579a45da-1f1a-4328-8e14-116468764cce job-eeef0bf6-d30d-4bed-a894-ff34adf42e80
job-eeef0bf6-d30d-4bed-a894-ff34adf42e80
Cannot kill job job-579a45da-1f1a-4328-8e14-116468764cce: Not enough permissions

```